pages/admin/index.tsx

```diff
@@ -1,16 +1,134 @@
 import { GetServerSideProps } from 'next'; // Get server side props
 import { getSession } from 'next-auth/client'; // Session management
-import { Text, GridItem } from '@chakra-ui/react'; // Chakra UI
+import {
+  Box,
+  Flex,
+  Text,
+  GridItem,
+  Button,
+  Tabs,
+  TabList,
+  Tab,
+  Menu,
+  MenuButton,
+  InputGroup,
+  Input,
+  InputLeftElement,
+} from '@chakra-ui/react'; // Chakra UI
+import { AddIcon, ChevronDownIcon, SearchIcon } from '@chakra-ui/icons'; // Chakra UI Icons
 import Layout from '@components/internal/Layout'; // Layout component
 import { Role } from '@lib/types'; // Role enum
 import { authorize } from '@tools/authorization'; // Page authorization
+import Table from '@components/internal/Table'; // Table component
+
+// Placeholder columns
+const COLUMNS = [
+  {
+    Header: 'Name',
+    accessor: 'name',
+  },
+  {
+    Header: 'Date Received',
+    accessor: 'dateReceived',
+  },
+  {
+    Header: 'Permit Type',
+    accessor: 'permitType',
+  },
+  {
+    Header: 'Request Type',
+    accessor: 'requestType',
+  },
+  {
+    Header: 'Status',
+    accessor: 'status',
+  },
+];
+
+// Placeholder data
+const DATA = [
+  {
+    name: 'Steve Rogers',
+    dateReceived: 'Dec 21 2021, 8:30 pm',
+    permitType: 'Permanent',
+    requestType: 'Replacement',
+    status: 'In progress',
+  },
+  {
+    name: 'Steve Rogers',
+    dateReceived: 'Dec 21 2021, 8:30 pm',
+    permitType: 'Permanent',
+    requestType: 'Replacement',
+    status: 'In progress',
+  },
+  {
+    name: 'Steve Rogers',
+    dateReceived: 'Dec 21 2021, 8:30 pm',
+    permitType: 'Permanent',
+    requestType: 'Replacement',
+    status: 'In progress',
+  },
+];
 
 // Internal home page - view APP requests
 export default function Requests() {
   return (
     <Layout>
       <GridItem colSpan={12} marginTop="64px">
-        <Text textStyle="display-xlarge">Internal management portal</Text>
+        <Flex justifyContent="space-between" alignItems="center" marginBottom="32px">
+          <Text textStyle="display-xlarge">Requests</Text>
+          <Button leftIcon={<AddIcon />} height="48px" padding="24px">
+            Create a request
+          </Button>
+        </Flex>
```

Comment:
Small nit: instead of `alignItems="center"` here, we should replace this with a `<Center />` component which comes bundled with Chakra.

Reply:
@jeffreyzhang2001 the `Center` component is a flexbox with `justifyContent=center` and `alignItems=center`, which won't work since we need `justifyContent=space-between`

Reply:
We can just use `Center` as well as `justifyContent=center`

Reply:
Resolved, we will stick with the current implementation as it is more explicit and we are not overriding hidden implementation.